source-jira/README.md

```diff
@@ -0,0 +1,91 @@
+# Jira source connector
+
+
+This is the repository for the Jira source connector, written in Python.
```

Comment:
General comment: We need to get rid of this file and others that are not applicable for our imported connector. For example, this airbyte-specific `README` is not useful for us to check in to our repository.

Reply:
I have a question: should I do this in a separated commit? If not, in which commit should I include it?

Reply:
I figured out that poetry demands a README.md file for each project. So we will need to keep but rewrite it.

Reply:
A separate commit would be fine for now.

Are you sure we need a README.md? Other connectors like [source-asana](https://github.com/estuary/connectors/tree/main/source-asana) don't have one.